services/submission/src/test/java/app/coronawarn/server/services/submission/SubmissionPayloadGenerator.java

```diff
@@ -0,0 +1,90 @@
+package app.coronawarn.server.services.submission;
+
+import app.coronawarn.server.common.protocols.external.exposurenotification.ReportType;
+import app.coronawarn.server.common.protocols.external.exposurenotification.TemporaryExposureKey;
+import app.coronawarn.server.common.protocols.internal.SubmissionPayload;
+
+import com.google.protobuf.ByteString;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class SubmissionPayloadGenerator {
+
+  public static final String MOBILE_CLIENT_PAYLOAD_PB_PATH =
+      "services/submission/src/test/resources/payload/mobile-client-payload.pb";
+
+  public static void main(String[] args) throws IOException {
+    int numberOfKeys = 10;
+    int transmissionRiskLevel = 6;
+    int rollingPeriod = 144; // 24*60/10
+    ReportType reportType = ReportType.CONFIRMED_CLINICAL_DIAGNOSIS;
+    ByteString requestPadding = ByteString.copyFrom(new byte[100]);
+    final List<String> visitedCountries = List.of("DE", "FR");
+    String originCountry = "DE";
+    boolean consentToFederation = true;
+    int daysSinceOnsetOfSymptoms = 0;
```

Comment:
Would propose to extract the variables as constants on class level for better readability

Reply:
or even better take some of the arguments as input parameter from `String[] args`